### ISSUE TITLE
Cannot run arch after installing appx - "The system cannot find the file specified."

### ISSUE THREAD

**Issue by MadCabbit** [yuk7/ArchWSL#136]
**Describe the issue**
The Arch shortcut just closes automatically.  Running arch from the cmd.exe gives "The system cannot find the file specified."  I was running Windows 10 1909 and upgraded to 2004 with no change.

**To Reproduce**
Run the Arch shortcut or arch from the command line.

**Expected behavior**
I expect it to actually launch.

**Enviroment:**
 - Windows build number: 10.0.19041.264
 - Security Software: [e.g. ESET Internet Security] Only Windows Defender.
 - WSL version 1/2: WSL2
 - ArchWSL version 20.4.3.0
 - ArchWSL Installer type: appx
 - Launcher version [e.g. 19030700] (Type `Arch.exe version` at a Command Prompt)
20040300

**Additional context**
I initially had another command "arch" in my path I thought was the culprit.  Since it belonged to an install I no longer use, I deleted said path, but repairing or reinstalling still gives the error.

**Comment by MadCabbit** [comment]
I decided to switch the default wsl to 2 and try reinstalling.  Now Arch refuses to reinstall on this machine (though it works in both wsl1 and wsl2 on my laptop).

Installing...
ERROR:Installation Failed!
HRESULT:0x80370109
Press any key to continue...

**Comment by MadCabbit** [comment]
The wslbridge error went away after a reboot, but the HRESULT error is still there.

**Comment by Lite5h4dow** [comment]
yeah im having the same issue but nothing will change or remove the error. I even reinstalled WSL 1 and 2and it didn't work with either.

**Comment by linuxmobile** [comment]
I've got same error installing with .cer, and appx. Any way to fix the error?

**Comment by CoderPat** [comment]
What solved it for me was unregistering Arch with (in CMD/Powershell):
```
wsl --unregister Arch
```

**Comment by liarchgh** [comment]
I've got the same error before.
Then I check the file named "rootfs.tar.gz", and I find that only Arch.exe is there and there is no rootfs.tar.gz in the same directory.
After I unzip rootfs.tar.gz from zip file and copy to the directory where Arch.exe is in, it works.

**Comment by noirgif** [comment]
Running `wsl --update`, unregister and rerun Arch.exe/installed package did solve the problem for me.
